extensions/amp-analytics/0.1/vendors/iwa.json

```diff
@@ -0,0 +1,13 @@
+{
+    "vars" : {
+        "tid": 1,
+        "appname": "amp",
+        "suid": "${clientId(_iwa_vid_main)}",
+        "pvid": "${pageViewId}"
+    },
+    "requests": {
+        "base": "//iwa3.hit.interia.pl:8443?json=",
+        "load": "${base}%7B%22LOAD%22%3A%7B%22Timestamp%22%3A${timestamp}%2C%22App_Name%22%3A%22amp%22%2C%22Msg_Type%22%3A%22LOAD%22%2C%22Can_Reload%22%3A1%2C%22Browser_Info%22%3A%22Timezone%3D${timesone}%26Resolution%3D${screenWidth}x${screenHeight}%26Colors%3D${screenColorDepth}%26View_Port%3D${viewportHeight}x${viewportWidth}%26Browser_Language%3D${browserLanguage}%22%2C%22Ver_Lib%22%3A%22amp%22%2C%22IFrame%22%3A0%7D%2C%22PAGE_VIEW%22%3A%7B%22Timestamp%22%3A${timestamp}%2C%22PageView_ID%22%3A%22${pvid}%22%2C%22Msg_Type%22%3A%22PAGE_VIEW%22%2C%22Service_UID%22%3A%22${suid}%22%2C%22Service_Host%22%3A%22${sh}%22%2C%22URL%22%3A%22${canonicalUrl}%22%2C%22Title%22%3A%22${pageTitle}%22%2C%22Refferer%22%3A%22${documentReferrer}%22%2C%22Encoding%22%3A%22${documentCharset}%22%2C%22PageView%22%3Atrue%2C%22Tracking_ID%22%3A${tid}%2C%22Request_Type%22%3A%22http%22%2C%22PageView_Data%22%3A%22gemiusId%253D${gemiusId}%2CarticleId%253D${articleId}%22%7D%7D",
```

Comment:
amp-analytics already encodes the request before it is sent out. Are you sure you want to manually encode the requests here?

Reply:
Yes, It needs to be encoded manually, because collector needs different format than encoded by amp-analytics